src/proposals/20230830/SparkGoerli_20230830.t.sol

```diff
@@ -0,0 +1,104 @@
+            wethConfigBefore.interestRateStrategy
+        );
+
+        _validateInterestRateStrategy(
+            address(interestRateStrategy),
+            address(interestRateStrategy),
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             OLD_WETH_OPTIMAL_USAGE_RATIO,
+                optimalStableToTotalDebtRatio: interestRateStrategy.OPTIMAL_STABLE_TO_TOTAL_DEBT_RATIO(),
+                baseStableBorrowRate:          OLD_WETH_VARIABLE_RATE_SLOPE_1,
+                stableRateSlope1:              interestRateStrategy.getStableRateSlope1(),
+                stableRateSlope2:              interestRateStrategy.getStableRateSlope2(),
+                baseVariableBorrowRate:        interestRateStrategy.getBaseVariableBorrowRate(),
+                variableRateSlope1:            OLD_WETH_VARIABLE_RATE_SLOPE_1,
+                variableRateSlope2:            OLD_WETH_VARIABLE_RATE_SLOPE_2
+            })
+        );
+        
+        /***********************/
+        /*** Execute Payload ***/
+        /***********************/
+        
+        GovHelpers.executePayload(vm, payload, executor);
+
+        ReserveConfig[] memory allConfigsAfter = createConfigurationSnapshot('', pool);
+        
+        /******************************************/
+        /*** wstETH Supply Cap After Assertions ***/
+        /******************************************/
+        
+        wstETHConfigBefore.supplyCap = NEW_WSTETH_SUPPLY_CAP;
+        _validateReserveConfig(wstETHConfigBefore, allConfigsAfter);
+        
+        /****************************************************/
+        /*** WETH Interest Rate Strategy After Assertions ***/
+        /****************************************************/
+
+        ReserveConfig memory wethConfigAfter = _findReserveConfigBySymbol(allConfigsAfter, 'WETH');
+
+        _validateInterestRateStrategy(
+            wethConfigAfter.interestRateStrategy,
+            wethConfigAfter.interestRateStrategy,
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             NEW_WETH_OPTIMAL_USAGE_RATIO,
+                optimalStableToTotalDebtRatio: interestRateStrategy.OPTIMAL_STABLE_TO_TOTAL_DEBT_RATIO(),
+                baseStableBorrowRate:          NEW_WETH_VARIABLE_RATE_SLOPE_1,
+                stableRateSlope1:              interestRateStrategy.getStableRateSlope1(),
+                stableRateSlope2:              interestRateStrategy.getStableRateSlope2(),
+                baseVariableBorrowRate:        interestRateStrategy.getBaseVariableBorrowRate(),
+                variableRateSlope1:            NEW_WETH_VARIABLE_RATE_SLOPE_1,
+                variableRateSlope2:            NEW_WETH_VARIABLE_RATE_SLOPE_2
+            })
+        );
+    }
+}
```

Comment:
Add new line at the end of file: https://stackoverflow.com/questions/44704968/visual-studio-code-insert-newline-at-the-end-of-files